bin/inference/pycbc_inference

```diff
@@ -151,6 +151,15 @@ with ctx:
     # construct class that will return the natural logarithm of likelihood
     model = models.read_from_config(cp, **model_args)
 
+    # Check if lower frequency cutoff on the command line and in config file are the same
+    if model._f_lower !=     low_frequency_cutoff_dict.values()[0]:
+        warnmsg = """Lower frequency cutoff {} specified in config file 
+        does not equal specification {} in command line argument""".format(
+            model._f_lower,     low_frequency_cutoff_dict.values()[0])
```

Comment:
Formatting the string like this will cause a newline and large space to be printed. However, if you wrap a string with parentheses, you can do new line continuation by just ending the quote and opening a new quote on the following line. So, change this to:
```
    warnmsg = ("Lower frequency cutoff {} specified in config file does not "
               "equal specification {} in command-line argument"
               .format(model._f_lower, low_frequency_cutoff_dict.values()[0])
```

Reply:
Wow, what happened there?!? My version looks damn ugly. Going to change that in a second.
If you do it like this, will the .format apply to both strings?

Reply:
Yes, the format applies to both.